### PR TITLE
Adds a extension in Int to convert into a timer string

### PR DESCRIPTION
**Pull request by RamonGilabert**
So I had this int going down and I was formatting it into a String but it was ugly as fuck since it was doing:

`14:9` instead of `14:09`.

So I did this.

### CODE REVIEW

**Comment by RamonGilabert** [comment]
@hyperoslo/malibu.

File: Source/Shared/Extensions/Int+Time.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+extension Int {
+
+  func time() -> String {
+    return self > 9 ? "\(self)" : "0\(self)"
```
Comment (onmyway133) [inline Source/Shared/Extensions/Int+Time.swift]: I think we can also `String(format: "%02d", self)`

**Comment by aashishdhawan** [comment]
@RamonGilabert @onmyway133 I would prefer not to add this function in `Sugar` or at least not in current form. I have few points i want to mention - 

1. We should add only those functions which will be generally used by developers and avoid using special utilities like this.
2. This just adds `0` to a number which is less than 9..too peculiar case.
3. The name of function `time()` is misleading (or at least ambiguous). I can not predict what this function does without looking into its implementation. 
4. `712.time()`..Hmm..looks odd to me.

**Comment by zenangst** [comment]
I agree, this is way beyond a normal use case. I'll go ahead and close this.

**Pull request closed without merging**